changelog.md

```diff
@@ -45,9 +45,11 @@
 - [FIXED] scope method syntax loses parameters when used multiple times [#7058](https://github.com/sequelize/sequelize/issues/7058)
 - [INTERNAL] Updated to `generic-pool@3.1.6` [#7109](https://github.com/sequelize/sequelize/issues/7109)
 - [FIXED] findAll throws error if attributes option is formatted incorrectly [#7162](https://github.com/sequelize/sequelize/issues/7163)
-- [FIXED] previous gave wrong value back [#7189](https://github.com/sequelize/sequelize/pull/7189)
+- [FIXED] model validation instance method now runs hooks by default [#7182](https://github.com/sequelize/sequelize/pull/7182)
+- [FIXED] previous gave wrong value back [#7189](https://github.com/sequelize/sequelize/issues/7178)
 
 ## BC breaks:
+- The model validation instance method now runs hooks by default. Previously, hooks would only run when explictly passed as follows: model.validate({hooks: true})
```

Comment:
`Model.validate` instance method run validation hooks by default now. Previously you need to pass `{ hooks: true }`. You can override this behavior by passing `{ hooks: false }`